doc/services/portability/posix/conformance/index.rst

```diff
@@ -0,0 +1,145 @@
+.. _posix_conformance:
+
+POSIX Conformance
+#################
+
+As per `IEEE 1003.1-2017`, this section details Zephyr's POSIX conformance.
+
+.. _posix_undefined_behaviour:
+
+.. note::
```

Comment:
If you make it a subheading instead of a note, and have the heading spelled "Undefined behavior" (it will arguably make it stand even more, and will make it more clear for folks why they landed there when clicking on one of the "unsupported" links), then you can `` :ref:`posix_undefined_behaviour` `` directly in your table and it will already have a meaningful and usable title

Reply:
Yeah, I'm not crazy about that standing out, to be honest.